src/components/member-profile/index.js

```diff
@@ -67,6 +71,10 @@ const Profile = (props) => {
     'linkedin_id',
     'instagram_id',
   ];
+  const { showMemberTaskUpdateModal, setIsOptionKey } = userContext();
+
+  const { query } = useRouter() || { query: { dev: false } };
```

Comment:
I feel we don't need to do this part ` || { query: { dev: false } }`

Reply:
okay I will remove it.